source/_includes/documentation-shell-integration.md

```diff
@@ -5,12 +5,12 @@ The easiest way to install shell integration is to select the *iTerm2&gt;Install
 
 When you select the *iTerm2&gt;Install Shell Integration* menu item, it types this for you:
 
-<pre>curl -L https://iterm2.com/misc/install_shell_integration.sh | bash</pre>
+<pre>curl -SsL https://iterm2.com/misc/install_shell_integration.sh | bash</pre>
```

Comment:
On 17.07.2016 01:06, George Nachman wrote:
>
> In source/_includes/documentation-shell-integration.md 
> <https://github.com/gnachman/iterm2-website/pull/31#discussion_r71072537>:
>
> > @@ -5,12 +5,12 @@ The easiest way to install shell integration is to select the *iTerm2&gt;Install
> >
> >  When you select the *iTerm2&gt;Install Shell Integration* menu item, it types this for you:
> >
> > -<pre>curl -L https://iterm2.com/misc/install_shell_integration.sh | bash</pre>
> > +<pre>curl -SsL https://iterm2.com/misc/install_shell_integration.sh | bash</pre>
>
> We send |curl -L|. Should we use -Ss for some reason? Seems useful to 
> get feedback that something is happening.
>

i added -s because the feedback is just progressbar, which usually 
clutters up terminal and in wrong place. the -S is added so the errors 
from curl are printed if the error occours. i don't find progressbar 
useful for such small downloads like <1k scripts.